simulations/vip-106/simulations.ts

```diff
@@ -0,0 +1,34 @@
+import { expect } from "chai";
+import { parseUnits } from "ethers/lib/utils";
+import { ethers } from "hardhat";
+
+import { forking, testVip } from "../../src/vip-framework";
+import { vip106 } from "../../vips/vip-106";
+import PAIR_ABI from "./abi/pairAbi.json";
+
+const PAIR = "0xD94FeFc80a7d10d4708b140c7210569061a7eddb";
+
+forking(26881099, () => {
+  let pair: ethers.Contract;
+  const provider = ethers.provider;
+  const currentLiquidityUsdt = parseUnits("90.294422022310547114", 18);
+  const currentLiquidityVai = parseUnits("87.950866165080484144", 18);
+
+  before(async () => {
+    pair = new ethers.Contract(PAIR, PAIR_ABI, provider);
+  });
+
+  testVip("VIP-106 Provide Liquidity in Pancake Swap", vip106());
+  describe("Post-VIP behavior", async () => {
```

Comment:
We can add another test to check the balance of the LP in the treasury, verifying it is increased after providing the liquidity

Reply:
Added [here](https://github.com/VenusProtocol/vips/pull/5/commits/248b385fc31cba942290dc719daf9149b17d2d4d)